frontends/mit-learn/src/pages/ChannelPage/ChannelPage.tsx

```diff
@@ -9,16 +9,64 @@ import type {
   BooleanFacets,
 } from "@mitodl/course-search-utils"
 import { ChannelTypeEnum } from "api/v0"
+import { useLearningResourceTopics } from "api/hooks/learningResources"
+import { ChipLink, Container, styled, Typography } from "ol-components"
+
+const SubTopicsContainer = styled(Container)(({ theme }) => ({
+  paddingTop: "50px",
```

Comment:
Did this get changed? I think it's 40px in figma now: https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=2296-49268&node-type=FRAME&m=dev.

The spacing above searchbar also looks too big to me (should be 64px)

Reply:
The top padding did indeed change, so I set it to 40px. As noted above, I'm not touching anything with the search box right now as there is an issue in the works to change that to be the new style we worked on for the home page. The spacing is built into the search input, and I didn't bother touching anything with that because it will just be undone when I go to work on the search input next.